src/3d/qgspostprocessingentity.cpp

```diff
@@ -0,0 +1,179 @@
+/***************************************************************************
+  qgspostprocessingentity.cpp
+  --------------------------------------
+  Date                 : August 2020
+  Copyright            : (C) 2020 by Belgacem Nedjima
+  Email                : gb underscore nedjima at esi dot dz
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "qgspostprocessingentity.h"
+
+#include <Qt3DRender/QGeometry>
+#include <Qt3DRender/QAttribute>
+#include <Qt3DRender/QBuffer>
+#include <Qt3DRender/QGeometryRenderer>
+#include <Qt3DRender/QParameter>
+#include <Qt3DRender/QTechnique>
+#include <Qt3DRender/QGraphicsApiFilter>
+#include <Qt3DRender/QDepthTest>
+
+#include "qgsshadowrenderingframegraph.h"
+
+QgsPostprocessingEntity::QgsPostprocessingEntity( QgsShadowRenderingFrameGraph *frameGraph, const QString &vertexShaderPath, const QString &fragmentShaderPath, QNode *parent )
+  : Qt3DCore::QEntity( parent )
+{
+  Qt3DRender::QGeometry *geom = new Qt3DRender::QGeometry( this );
+  Qt3DRender::QAttribute *positionAttribute = new Qt3DRender::QAttribute( this );
+  QVector<float> vert =
+  {
+    -1.0f, -1.0f, 0.0f,
+      1.0f, -1.0f, 0.0f,
+      -1.0f,  1.0f, 0.0f,
+      -1.0f,  1.0f, 0.0f,
+      1.0f, -1.0f, 0.0f,
+      1.0f,  1.0f, 0.0f
+    };
```

Comment:
can you adjust formatting of the block so it is nicely aligned?

Reply:
I adjusted them correctly at first, but the prepare_commit script turns them like this.